package.json

```diff
@@ -22,34 +26,34 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@holochain/conductor-api": "0.0.1-dev.10",
+    "@holochain/conductor-api": "holochain/holochain-conductor-api#tester-branch",
```

Comment:
@maackle or @zo-el if you publish a new holochain-conductor-api then it can be pointed to here

Reply:
done